chore: Add env setup instructions to README (#180)
techmatters/terraso-mobile-client: README.md

```
jarsigner -verbose -sigalg SHA256withRSA -digestalg SHA-256  -keystore ~/terraso-lpks-key.keystore -signedjar app/build/outputs/bundle/release/terraso-landpks.aab  app/build/outputs/bundle/release/app-release.aab terraso-lpks
```

# Env Setup

## Logging In

> Once all of the dependencies are installed, the first barrier to running the project locally is getting past the login screen. In order to do so, you'll need to do the following:

Reviewer: ```suggestion
Once all of the dependencies are installed, you'll need to do the following to enable logins:
```

1. Set up a Google OAuth project for Android or iOS if it doesn't already exist
- See [Google Cloud Console](https://console.cloud.google.com/)

> 2. Copy `.env.sample` to `.env`. You need to change the `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match the value of your OAuth App client id

Reviewer: ```suggestion
2. Copy `.env.sample` to `.env`. Change `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match the value of your OAuth App client ID in Google Cloud Console.
```

> 2. Copy `.env.sample` to `.env`. You need to change the `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match

Reviewer: Please clarify `change to match`.
